src/order/disjoint.lean

```diff
@@ -242,6 +242,9 @@ lemma codisjoint.left_le_of_le_inf_right (h : a ⊓ b ≤ c) (hd : codisjoint b
 lemma codisjoint.left_le_of_le_inf_left (h : b ⊓ a ≤ c) (hd : codisjoint b c) : a ≤ c :=
 hd.left_le_of_le_inf_right $ by rwa inf_comm
 
+lemma codisjoint.inf_left_sup_inf_right (a : α) (h : codisjoint b c) : a ⊓ b ⊔ a ⊓ c = a :=
```

Comment:
Isn't this a trivial consequence of `inf_sup_left`?

Reply:
Removed.